### PR TITLE
Setup.py

### LINKED ISSUE #8: Going to get started on standard documentation / command line capabilities next
I finished up making the changes to the setup.py file. The standard documentation does not look too complicated so I think I will go ahead and add that in as well. After that, I will get started on the command line functions for the package!

### PR DESCRIPTION
**Pull request by patrickmorrell17**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from distutils.core import setup
+setup(
+  name = 'PLFA_Tools_Aggregator',         # How you named your package folder (MyLib)
```
Comment (bryanrcarlson) [inline setup.py]: `name = 'plfatools'`

Users can use: `pip install plfatools`

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  packages = ['PLFA_Tools_Aggregator'],   # Chose the same as "name"
```
Comment (bryanrcarlson) [inline setup.py]: I think this is `packages = ['pflatools']` as well

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  version = '0.0.1',      # Start with a small number and increase it with every change you make
```
Comment (bryanrcarlson) [inline setup.py]: Lets start at `version = '0.1.0'`. I think 0.0.1 suggests alpha (internal only). I think we're more like beta (external use, but with extreme caution).

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  license='CC0',        # Chose a license from here: https://help.github.com/articles/licensing-a-repository
+  description = 'The purpose of this package is to automate the aggregation of the data into a single csv file for analysis.',   # Give a short description about your library
+  author = 'Patrick Morrell',                   # Type in your name
```
Comment (bryanrcarlson) [inline setup.py]: `author = 'Patrick Morrell, Bryan Carlson'`

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  author_email = 'bryan.carlson@usda.gov',      # Type in your E-Mail
+  url = 'https://github.com/cafltar/plfa_tools_aggregator',   # Provide either the link to your github or to your website
+  download_url = 'https://github.com/cafltar/plfa_tools_aggregator.git',    # I explain this later on
```
Comment (bryanrcarlson) [inline setup.py]: We'll need to update this later to the release commit, once we finalize and tag it

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  keywords = ['Aggregator', 'PLFA', 'CSV'],   # Keywords that define your package best
+  install_requires=  ['pathlib', 'pandas', 'numpy'],
+  classifiers=[
+    'Development Status :: 3 - Alpha',      # Chose either "3 - Alpha", "4 - Beta" or "5 - Production/Stable" as the current state of your package
```
Comment (bryanrcarlson) [inline setup.py]: I think 4 - Beta?

**Review by bryanrcarlson — CHANGES_REQUESTED** [review]
Looking good! Some details need updated, then I think it's ready for merge

**Comment by patrickmorrell17** [comment]
Ok- I will make these changes and re-push the branch! Do you think I need to delete the comments on the setup.py?

**Comment by bryanrcarlson** [comment]
@patrickmorrell17 , yeah, feel free to remove the comments in setup.py. Thanks!

**Review by bryanrcarlson — APPROVED** [review]

**Pull request merged**